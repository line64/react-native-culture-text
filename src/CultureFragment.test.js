import "react-native";
import React from "react";
import { View, TextInput } from "react-native";
import { CultureContext, CultureFragment } from "../";

import renderer from "react-test-renderer";

const translations = {
  es: {
    placeholder: "indicio de un campo de texto"
  }
};

it("render translated message", () => {
  const tree = renderer
    .create(
      <CultureContext translations={translations} locale="es">
        <View>
          <CultureFragment
            children={T => (
              <TextInput palceholder={T("placeholder")}/>
            )}
          />
        </View>
      </CultureContext>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
