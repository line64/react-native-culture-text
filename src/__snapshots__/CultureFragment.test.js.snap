// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render translated message 1`] = `
<View>
  <TextInput
    allowFontScaling={true}
    palceholder="indicio de un campo de texto"
  />
</View>
`;
